client/extensions/woocommerce/app/product-categories/list.js

```diff
@@ -0,0 +1,214 @@
+	};
+
+	getRowHeight = ( { index } ) => {
+		return this.getItemHeight( this.getItem( index ) );
+	};
+
+	getItem( index ) {
+		if ( this.props.categories ) {
+			return this.props.categories[ index ];
+		}
+	}
+
+	renderItem( item, _recurse = false ) {
+		const { site } = this.props;
+
+		// if item has a parent and it is in current props.categories, do not render
+		if ( item.parent && ! _recurse && includes( this.catIds, item.parent ) ) {
+			return;
+		}
+		const children = this.getChildren( item.id );
+		const itemId = item.id;
+		const image = item.image && item.image.src;
+		const imageClasses = classNames( 'product-categories__list-item-icon', {
+			'is-thumb-placeholder': ! image,
+		} );
+		const link = getLink( '/store/products/category/:site/' + itemId, site );
+
+		const goToLink = () => {
+			page( link );
+		};
+
+		return (
+			<div key={ 'product-category-' + itemId } className="product-categories__list-item">
+				<CompactCard key={ itemId } className="product-categories__list-item-card" onClick={ goToLink }>
+					<div className="product-categories__list-item-wrapper">
+						<div className={ imageClasses }>{ item.image && <img src={ item.image.src } /> }</div>
+						<span className="product-categories__list-item-info">
+							<a href={ link }>{ item.name }</a>
+							<Count count={ item.count } />
+							<span className="product-categories__list-item-description">{ item.description }</span>
+						</span>
+					</div>
+				</CompactCard>
+					{ children.length > 0 && (
+						<div className="product-categories__list-nested">
+							{ children.map( child => this.renderItem( child, true ) ) }
+						</div>
+					) }
+			</div>
+		);
+	}
+
+	renderRow = ( { index } ) => {
+		const item = this.getItem( index );
+		if ( item ) {
+			return this.renderItem( item );
+		}
+
+		return null;
+	};
+
+	renderCategoryList() {
+		const { loading, categories, lastPage, searchQuery } = this.props;
+		return (
+			<WindowScroller>
+				{( { height, scrollTop } ) => (
+					<VirtualList
+						items={ categories }
+						lastPage={ lastPage }
+						loading={ loading }
+						getRowHeight={ this.getRowHeight }
+						renderRow={ this.renderRow }
+						onRequestPages={ this.props.requestPages }
+						perPage={ DEFAULT_PRODUCT_CATEGORIES_PER_PAGE }
+						loadOffset={ 10 }
+						searching={ searchQuery && searchQuery.length }
+						defaultRowHeight={ ITEM_HEIGHT }
+						height={ height }
+						scrollTop={ scrollTop }
+					/>
+				)}
+			</WindowScroller>
+		);
+	}
+
+	render() {
+		const { className, translate, totalCategories, searchQuery } = this.props;
+
+		if ( this.props.isInitialRequestLoaded && 0 === totalCategories ) {
+			let message = null;
+			if ( searchQuery ) {
+				message = translate( 'No product categories found for {{query /}}.', {
+					components: {
+						query: <em>{ searchQuery }</em>,
+					},
+				} );
+			}
+			return <EmptyContent title={ translate( 'No product categories found.' ) } line={ message } />;
```

Comment:
Hi! I've found a possible matching string that has already been translated **92** times:
`translate( 'No categories found.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).